backend/app/core/exception_handlers.py

```
@@ -0,0 +1,33 @@
+            content=APIResponse.failure_response(exc.errors()).model_dump(),
+        )
+
+    @app.exception_handler(HTTPException)
+    async def http_exception_handler(request: Request, exc: HTTPException):
+        return JSONResponse(
+            status_code=exc.status_code,
+            content=APIResponse.failure_response(exc.detail).model_dump(),
+        )
+
+    @app.exception_handler(Exception)
+    async def generic_error_handler(request: Request, exc: Exception):
```

Ruff (0.14.10)

Unused function argument: `request`

(ARG001)